index.html

```diff
@@ -211,5 +211,141 @@ <h3 class="text header-3">How it works</h3>
+                  </p>
+                </div>
+              </div>
+              <p class="text body-small">
+                The best online zoo I’ve met. My son delighted very much ljves
+                to watch gouillas. Online zoo is one jf the ways to instill a
+                love for animals.<br />
+                The best online zoo I’ve met. My son delighted very much ljves
+                to watch gouillas. Online zoo is one jf the ways to instill a
+                love for animals. <br />
+                The best online zoo I’ve met. My son delighted very much ljves
+                to watch gouillas. Online zoo is one jf the ways to instill a
+                love for animals.The best online zoo I’ve met. <br />
+                The best online zoo I’ve met. My son delighted very much ljves
+                to watch gouillas. Online zoo is one jf the ways to instill a
+                love for animals.<br />
+                The best online zoo I’ve met. My son delighted very much ljves
+                to watch gouillas. Online zoo is one jf the ways to instill a
+                love for animals.
+              </p>
+            </div>
+          </div>
+        </div>
+        <div style="height: 391px">
+          <div class="gradient">
+            <div class="testimonial-card">
+              <div class="header">
+                <img src="./assets/icons/ellipse_foto_4.svg" />
+                <div class="texts">
+                  <p class="text body">Mila Riksha</p>
+                  <p class="text body-small">
+                    Local Austria <span>.</span> Yesterday
+                  </p>
+                </div>
+              </div>
+              <p class="text body-small">
+                My son delighted very much ljves to watch gouillas. Online zoo
+                is one jf the ways to instill a love for animals.<br />
+                The best online zoo I’ve met. My son delighted very much ljves
+                to watch gouillas. Online zoo is one jf the ways to instill a
+                love for animals.<br />
+                The best online zoo I’ve met. My son delighted very much ljves
+                to watch gouillas. Online zoo is one jf the ways to instill a
+                love for animals.<br />
+                The best online zoo I’ve met. My son delighted very much ljves
+                to watch gouillas.<br />
+                The best online zoo I’ve met. My son delighted very much ljves
+                to watch gouillas. Online zoo is one jf the ways to instill a
+                love for animals.<br />
+                The best online zoo I’ve met. My son delighted very much ljves
+                to watch gouillas. Online zoo is one jf the ways to instill a
+                love for animals.
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="indicator" style="width: 599px; height: 6px">
+        <div class="background">
```

Comment:
Yes, I need this div, because I use for indicator style gradient-back, and inside this class I have another div for my content (it's a way to make a gradient border, I have no another way).